pasta.html

```diff
@@ -59,10 +59,9 @@ <h2>Steps to cook Chicken Pasta</h2>
             <li>Serve hot with added garnishing of shredded cheese on the top.</li>
         </ol>
         <p class="video">For the summary of how to cook recipe, the video is attached.</p>
-        <div>
+        <div class="videoplay">
             <iframe width="560" height="315" src="https://www.youtube.com/embed/esapRRrai3o" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
         </div>
-        <a href="intro.html">Return to main menu</a>
     </div>
```

Comment:
Please fix the linter error by giving additional space. You may use [this link](https://github.com/microverseinc/linters-config) to fix all linter errors.

Reply:
- [ ] As I commented above your comment should be phrased politely, as suggestions instead of orders, and always tell the reason WHY. :+1:  Please fix it :+1: